backend/FwLite/FwLiteProjectSync/CrdtFwdataProjectSyncService.cs

```diff
@@ -176,6 +180,32 @@ private async Task<int> ExampleSentenceSync(Guid entryId,
             });
     }
 
+    private async Task<int> PartOfSpeechSync(PartOfSpeech[] currentEntries,
+        PartOfSpeech[] previousEntries,
+        IMiniLcmApi api)
+    {
+        return await DiffCollection(api,
+            previousEntries,
+            currentEntries,
+            async (api, currentPos) =>
+            {
+                await api.CreatePartOfSpeech(currentPos);
+                return 1;
+            },
+            async (api, previousPos) =>
+            {
+                await api.DeletePartOfSpeech(previousPos.Id);
+                return 1;
+            },
+            async (api, previousPos, currentPos) =>
+            {
+                var updateObjectInput = PartOfSpeechDiffToUpdate(previousPos, currentPos);
+                if (updateObjectInput is not null) await api.UpdatePartOfSpeech(currentPos.Id, updateObjectInput);
+                var changes = 0; // await SenseSync(currentPos.Id, currentPos.Senses, previousPos.Senses, api);
+                return changes + (updateObjectInput is null ? 0 : 1);
```

Comment:
this could be cleaned up, get rid of the `changes` variable

Reply:
Done in commit 9680e530.